lib/shared/addon/utils/util.js

```diff
@@ -436,7 +436,7 @@ function keyNotType(k) {
 
 export function removeEmpty(obj, excludedKeys = []){
   return Object.keys(obj)
-    .filter((k) => !isEmpty(obj[k]) && (typeof obj[k] !== 'object' || keyNotType(obj[k])))
+    .filter((k) => excludedKeys.indexOf(k) !== -1 || (!isEmpty(obj[k]) && (typeof obj[k] !== 'object' || keyNotType(obj[k]))))
```

Comment:
There are still places using indexOf / -1 but most checks have been changed to `excludedKeys.includes(k)` because it's easier to read and not get wrong.

Reply:
Fixed and updated the other existing indexOf reference in the method.